src/test/scala/uk/gov/hmrc/perftests/SCA/SCAUserJourney.scala

```diff
@@ -0,0 +1,116 @@
+      .formParam("agent.agentId", "")
+      .formParam("agent.agentCode", "")
+      .formParam("agent.agentFriendlyName", "")
+      .formParam("unreadMessageCount", "")
+      .formParam("mdtp.sessionId", "")
+      .formParam("mdtp.deviceId", "")
+      .formParam("presets-dropdown", "IR-SA")
+      .formParam("enrolment[0].name", "")
+      .formParam("enrolment[0].taxIdentifier[0].name", "")
+      .formParam("enrolment[0].taxIdentifier[0].value", "")
+      .formParam("enrolment[0].state", "Activated")
+      .formParam("enrolment[1].name", "")
+      .formParam("enrolment[1].taxIdentifier[0].name", "")
+      .formParam("enrolment[1].taxIdentifier[0].value", "")
+      .formParam("enrolment[1].state", "Activated")
+      .formParam("enrolment[2].name", "")
+      .formParam("enrolment[2].taxIdentifier[0].name", "")
+      .formParam("enrolment[2].taxIdentifier[0].value", "")
+      .formParam("enrolment[2].state", "Activated")
+      .formParam("enrolment[3].name", "")
+      .formParam("enrolment[3].taxIdentifier[0].name", "")
+      .formParam("enrolment[3].taxIdentifier[0].value", "")
+      .formParam("enrolment[3].state", "Activated")
+      .formParam("enrolment[4].name", "IR-SA")
+      .formParam("enrolment[4].taxIdentifier[0].name", "UTR")
+      .formParam("enrolment[4].taxIdentifier[0].value", "1632631936")
+      .formParam("enrolment[4].state", "Activated")
+      .formParam("itmp.givenName", "")
+      .formParam("itmp.middleName", "")
+      .formParam("itmp.familyName", "")
+      .formParam("itmp.dateOfBirth", "")
+      .formParam("itmp.address.line1", "")
+      .formParam("itmp.address.line2", "")
+      .formParam("itmp.address.line3", "")
+      .formParam("itmp.address.line4", "")
+      .formParam("itmp.address.line5", "")
+      .formParam("itmp.address.postCode", "")
+      .formParam("itmp.address.countryName", "")
+      .formParam("itmp.address.countryCode", "")
+      .check(status.is(303))
+      .check(currentLocation.is(GGLoginFrontEnd + "/auth-login-stub/gg-sign-in": String))
+  }
+
+  // SCA Home Page
+  def getSCAHomePage: HttpRequestBuilder =
+    http("GET - SCA Home page")
+      .get(SCAFrontEnd + "/single-customer-account": String)
+      .check(status.in(200, 303))
+      .check(currentLocation.is(SCAFrontEnd + "/single-customer-account": String))
+
+  def getYourTaxesAndBenefits: HttpRequestBuilder =
+    http("GET - YourTaxesAndBenefits")
+      .get(YourTaxesAndBenefits + "/single-customer-account/taxes-and-benefits": String)
+      .check(status.in(200))
+      .check(currentLocation.is(YourTaxesAndBenefits + "/single-customer-account/taxes-and-benefits": String))
+
+  def getMessages: HttpRequestBuilder =
+    http("GET - SCA Messages")
+      .get(Messages + "/single-customer-account/messages": String)
+      .check(status.in(200))
+      .check(currentLocation.is(Messages + "/single-customer-account/messages": String))
+
+  def getFeedbackPage: HttpRequestBuilder =
```

Comment:
we don't need to perf test the feedback service

Reply:
Actioned